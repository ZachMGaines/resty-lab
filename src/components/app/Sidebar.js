import React from 'react';


const Sidebar = () => {
  return (
    <ul id="side">
      <h2>Sidebar Stuff</h2>
    </ul>
  );
};

export default Sidebar;
