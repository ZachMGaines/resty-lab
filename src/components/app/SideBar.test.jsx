import React from 'react';
import { render } from '@testing-library/react';
import Sidebar from './Sidebar';

describe('Sidebar component', () => {
  it('renders ul', () => {
    const { asFragment } = render(
      <Sidebar />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
