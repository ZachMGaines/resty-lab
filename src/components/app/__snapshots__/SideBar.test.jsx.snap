// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component renders ul 1`] = `
<DocumentFragment>
  <ul
    id="side"
  >
    <h2>
      Sidebar Stuff
    </h2>
  </ul>
</DocumentFragment>
`;
